packages/charts/react-charting/src/components/HeatMapChart/__snapshots__/HeatMapChartRTL.test.tsx.snap

```diff
@@ -1410,132 +1410,56 @@ exports[`HeatMapChart snapshot tests should render axis labels correctly When cu
           </g>
         </g>
         <g
-          aria-label="xPoint p3, yPoint p2. Nasty, 25."
+          aria-label="xPoint p3, yPoint p2. , No data available."
```

Comment:
>data [](http://example.com/codeflow?start=49&length=4)

why this change

Reply:
need to see, I just updated snapshots so that build passes.